templates/browse/row.inc

```diff
@@ -72,11 +72,11 @@ for ($c = 1; $c <= count($this->columns); $c++) {
         if ($type == 'email') {
             $this->variables[$this->columns[$c - 1]] =
                 new Horde_Form_Variable('', $this->columns[$c - 1],
-                                        Horde_Form::getType('html'), false);
+                                        (new Horde_Form(0))->getType('html'), false);
```

Comment:
https://dev.horde.org/api/FRAMEWORK_4/lib/Form/source-class-Horde_Form.html#151-171

That method is awful. Thank you for the patch.